packages/plugin-conversation/test/integration/spec/verbs.js

```diff
@@ -337,6 +338,34 @@ describe(`plugin-conversation`, function() {
         }));
     });
 
+    describe(`#updateTypingStatus()`, () => {
+      beforeEach(() => spark.conversation.create({participants, comment: `THIS IS A COMMENT`})
+        .then((c) => {conversation = c;}));
+
+      it(`sets the typing indicator for the specified conversation`, () => {
+        const spy = sinon.spy();
+        mccoy.spark.mercury.on(`event:status.start_typing`, spy);
+
+        return spark.conversation.updateTypingStatus(conversation, {typing: true})
+        .then(() => spark.conversation.get(conversation))
```

Comment:
Gotcha. That is exactly what I was using it for because it wouldn't receive the mercury event before the test ended otherwise.